doc/modules/mod_inbox.md

```diff
@@ -34,36 +34,37 @@ Inbox currently supports the following DBs:
 Alice sends:
 
 <message type="chat" to="bOb@localhost/res1" id=”123”>
-<body>Hello</body>
+  <body>Hello</body>
 </message>
 
 Bob receives:
 
 <message from="alicE@localhost/res1" to="bOb@localhost/res1" id=“123” xml:lang="en" type="chat">
-<body>Hello</body>
+  <body>Hello</body>
 </message>
 
 Alice sends:
 
 <iq type="get" id="10bca">
-<inbox xmlns=”erlang-solutions.com:xmpp:inbox:0” queryid="b6"/>
+  <inbox xmlns=”erlang-solutions.com:xmpp:inbox:0” queryid="b6"/>
 </iq>
 
 
 Alice receives:
 
 <message from="alicE@localhost" to="alicE@localhost" id="9b759">
-<result xmlns=”erlang-solutions.com:xmpp:inbox:0” unread="0" queryid="b6">
-<forwarded xmlns=”urn:xmpp:forward:0”>
-<message xml:lang="en" type="chat" to="bOb@localhost/res1" from="alicE@localhost/res1" id=”123”>
-<body>Hello</body>
-</message>
-</forwarded>
-</result>
+  <result xmlns="erlang-solutions.com:xmpp:inbox:0" unread="0" queryid="b6">
+    <forwarded xmlns="urn:xmpp:forward:0">
```

Comment:
I wonder why there is the `urn:xmpp:forward:0` namespace used instead of `erlang-solutions.com:xmpp:inbox:0`. That's just a question.

Reply:
https://xmpp.org/extensions/xep-0297.html

Reply:
Thanks @ludwikbukowski, this makes sense!